### PR TITLE
Update native integers conversions

### PR DESCRIPTION
**Pull request by Youssef1313**
Replaces https://github.com/dotnet/csharplang/pull/3938

### CODE REVIEW

File: proposals/csharp-9.0/native-integers.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ The tables below cover the conversions between special types.
 | `decimal` | `nuint` | ExplicitNumeric | `ulong decimal.op_Explicit(decimal) conv.u` / `... conv.ovf.u.un`  |
 | `IntPtr` | `nuint` | None | |
 | `UIntPtr` | `nuint` | Identity | |
+|Enumeration|`nint`|ExplicitEnumeration||
+|Enumeration|`nuint`|ExplicitEnumeration||
```
Comment (Youssef1313) [inline proposals/csharp-9.0/native-integers.md:81]: I'm unsure of the IL, so left them empty for now until someone confirms the correct IL.

File: proposals/csharp-9.0/native-integers.md [hunk 2]
```diff
@@ -353,8 +357,8 @@ But the language would not provide keywords, constants, or `checked` operations.
...
-https://github.com/dotnet/csharplang/blob/master/meetings/2017/LDM-2017-05-26.md
-https://github.com/dotnet/csharplang/blob/master/meetings/2017/LDM-2017-06-13.md
-https://github.com/dotnet/csharplang/blob/master/meetings/2017/LDM-2017-07-05.md#native-int-and-intptr-operators
-https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-10-23.md
-https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-03-25.md
+- https://github.com/dotnet/csharplang/blob/master/meetings/2017/LDM-2017-05-26.md
+- https://github.com/dotnet/csharplang/blob/master/meetings/2017/LDM-2017-06-13.md
+- https://github.com/dotnet/csharplang/blob/master/meetings/2017/LDM-2017-07-05.md#native-int-and-intptr-operators
+- https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-10-23.md
+- https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-03-25.md
```
Comment (Youssef1313) [inline proposals/csharp-9.0/native-integers.md:364]: Before this change, links used to appear in the same line

**Review by cston — APPROVED** [review]

**Review by jcouv — APPROVED** [review]

File: proposals/csharp-9.0/native-integers.md (hunk 1 above)
Comment (Youssef1313) [reply proposals/csharp-9.0/native-integers.md:81]: @jcouv @cston, the table is still missing the IL instructions in the last column

Comment (cston) [reply proposals/csharp-9.0/native-integers.md:81]: The IL depends on the underlying type of the `enum` so it seems fine to leave the column blank.

**Pull request merged**